project/CONTRIBUTING.md

```diff
@@ -271,6 +271,18 @@ or postponed to next major release if BC is not possible.
 
 If you are not sure what should be done, don't hesitate to open an issue about your PR project.
 
+##### Dependencies changes
```

Comment:
"Dependency changes"

Reply:
Well, you don't ever made changes about only one dependency, right?

Reply:
You make several dependency changes

Reply:
Indeed.